skylark_library.bzl

```diff
@@ -32,9 +32,8 @@ def _skylark_library_impl(ctx):
         # a separate program, or from `tools` of a genrule().
         DefaultInfo(
             files = all_files,
-            runfiles = ctx.runfiles(files = list(all_files)),
+            runfiles = ctx.runfiles(files = all_files.to_list()),
```

Comment:
I don't think `to_list` is needed here.

Reply:
Removing this change with `--all_incompatible_changes` enabled produces this error on `bazel build ...`

```
ERROR: /home/james/opensource/bazel-skylib/lib/BUILD:53:1: in skylark_library rule //lib:structs: 
Traceback (most recent call last):
	File "/home/james/opensource/bazel-skylib/lib/BUILD", line 53
		skylark_library(name = 'structs')
	File "/home/james/opensource/bazel-skylib/skylark_library.bzl", line 33, in _skylark_library_impl
		DefaultInfo(files = all_files, runfiles = ctx....)))
	File "/home/james/opensource/bazel-skylib/skylark_library.bzl", line 35, in DefaultInfo
		ctx.runfiles(files = list(all_files))
	File "/home/james/opensource/bazel-skylib/skylark_library.bzl", line 35, in ctx.runfiles
		list(all_files)
type 'depset' is not iterable. Use the `to_list()` method to get a list. Use --incompatible_depset_is_not_iterable=false to temporarily disable this check.
ERROR: Analysis of target '//lib:structs' failed; build aborted: Analysis of target '//lib:structs' failed; build aborted
```

Reply:
What about this?
`            runfiles = ctx.runfiles(files = all_files),`